hvpy/facade.py

```diff
@@ -27,3 +27,106 @@ def getJP2Image(
+    """
+    params = getJP2HeaderInputParameters(id=id, callback=callback)
+    return execute_api_call(input_parameters=params)
+
+
+@add_shared_docstring(getJPXClosestToMidPointInputParameters)
+def getJPXClosestToMidPoint(
+    startTimes: List[datetime],
+    endTimes: List[datetime],
+    sourceId: int,
+    linked: bool = True,
+    verbose: bool = False,
+    jpip: bool = False,
+) -> Union[bytes, str, Dict[str, Any]]:
+    """
+    Generate and (optionally) download a custom JPX movie of the specified
+    datasource with one frame per pair of startTimes/endTimes parameters.
+
+    {Insert}
+    Examples
+    --------
+    >>> from hvpy import getJPXClosestToMidPoint
+    >>> from datetime import datetime
+    >>> getJPXClosestToMidPoint(startTimes=[datetime(2014, 1, 1, 0, 0, 0), datetime(2014, 1, 1, 2, 3, 3)], endTimes=[datetime(2014, 1, 1, 0, 45, 0), datetime(2014, 1, 1, 2, 33, 3)], sourceId=14, linked=False, jpip=True)
+    'jpip://helioviewer.org:8090/movies/SDO_AIA_335_F2013-12-31T18.30.00Z_T2013-12-31T21.03.03ZCMP.jpxmid'
+    """
+    params = getJPXClosestToMidPointInputParameters(
+        startTimes=startTimes,
+        endTimes=endTimes,
+        sourceId=sourceId,
+        linked=linked,
+        verbose=verbose,
+        jpip=jpip,
+    )
+    return execute_api_call(input_parameters=params)
+
+
+@add_shared_docstring(getJPXInputParameters)
+def getJPX(
+    startTime: List[datetime],
+    endTime: List[datetime],
+    sourceId: int,
+    linked: bool = True,
+    verbose: bool = False,
+    jpip: bool = False,
+    cadence: Optional[int] = None,
+) -> Union[bytes, str, Dict[str, Any]]:
+    """
+    Generate and (optionally) download a custom JPX movie of the specified
+    datasource from the helioviewer.org API.
+
+    {Insert}
+    Examples
+    --------
+    >>> from hvpy import getJPX
+    >>> from datetime import datetime
+    >>> getJPX(startTime=datetime(2014, 1, 1, 0, 0, 0),endTime=datetime(2014, 1, 1, 0, 45, 0),sourceId=14,jpip=True)
+    'jpip://helioviewer.org:8090/movies/SDO_AIA_335_F2014-01-01T00.00.00Z_T2014-01-01T00.45.00ZL.jpx'
+    """
+    params = getJPXInputParameters(
+        startTime=startTime,
+        endTime=endTime,
+        sourceId=sourceId,
+        linked=linked,
+        verbose=verbose,
+        jpip=jpip,
+        cadence=cadence,
+    )
+    return execute_api_call(input_parameters=params)
+
+
+@add_shared_docstring(getStatusInputParameters)
+def getStatus() -> Union[bytes, str, Dict[str, Any]]:
+    """
+    Returns information about how far behind the latest available JPEG2000
+    images.
+
+    {Insert}
+    Examples
+    --------
+    >>> from hvpy import getStatus
+    >>> getStatus()
+    {'AIA': {'time': '2022-07-11T15:35:26Z', 'level': 1, 'secondsBehind': 2525, 'measurement': 'AIA 1600'}, ... 'XRT': {'time': '2022-06-24T23:56:45Z', 'level': 5, 'secondsBehind': 1441246, 'measurement': 'XRT Any/Any'}}
```

Comment:
I think these numbers are going to change everytime you run the status, you might want to just add `...` for the ones that do.